docs/pages/archived/managed-workflow.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Managed workflow
+---
+
+> TL;DR: "Managed workflow" is now just referred to as "Using Expo Prebuild".
+
+Up until SDK 41, the terms **managed workflow** and **bare workflow** had a completely different set of limitations than they do today, and developers had to choose which set of features they wanted to use.
```

Comment:
```suggestion
Until SDK 41, the terms **managed workflow** and **bare workflow** had a completely different set of limitations than they do today, and developers had to choose which set of features they wanted to use.
```

I think using the word `Until` is enough to provide enough clarity since the SDK version is mentioned.